### PR TITLE
Add documentation for the variadic parameters example

### LINKED ISSUE #59: Add Document for Variadic Parameter Program
Refer below commits:
- [Commit 1](https://github.com/sagar-jadhav/go-examples/commit/1522697baa28bae38b964307b1adc353ff3eab20)
- [Commit 2](https://github.com/sagar-jadhav/go-examples/commit/1522697baa28bae38b964307b1adc353ff3eab20)
- [Commit 3](https://github.com/sagar-jadhav/go-examples/commit/f47deddb11788383563ff439008bf37f333e92bd)

### PR DESCRIPTION
**Pull request by aggressivepixels**
Fixes #59

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ To perform hands-on on **go-examples** it is prerequisite to have Go installed o
 * [Struct](https://github.com/sagar-jadhav/go-examples/blob/master/src/struct.go)
 * [Switch Statement](https://github.com/sagar-jadhav/go-examples/blob/master/src/switch.go)
 * [Unit Test](https://github.com/sagar-jadhav/go-examples/tree/master/src/test/power_test.go)
-* [Variadic Parameter](https://github.com/sagar-jadhav/go-examples/blob/master/src/variadic.go)
+* [Variadic Parameter](./variadic.md)
```
Comment (sagar-jadhav) [inline README.md]: It should be `./variadic.html`

File: variadic.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# Variadic functions
+
+The example demonstrates variadic functions. Variadic functions are functions which accept a variable number of parameters.
+
+```
+package main
+
+import "fmt"
+
+/*
```
Comment (sagar-jadhav) [inline variadic.md]: Remove comments from here as it is already mentioned in description

**Review by sagar-jadhav — CHANGES_REQUESTED** [review]

File: variadic.md (hunk 2 above)
Comment (aggressivepixels) [reply variadic.md]: Should the single-line comments in `main` be removed too?

Comment (sagar-jadhav) [reply variadic.md]: No let them as it is

**Review by sagar-jadhav — APPROVED** [review]

**Pull request merged**